pkg/harvester/edit/harvesterhci.io.storage/provisioners/driver.longhorn.io_v1.vue

```diff
@@ -141,11 +130,17 @@ export default {
       get() {
         const parameters = clone(this.value?.parameters) || {};
 
-        DEFAULT_PARAMETERS.forEach((key) => {
-          delete parameters[key];
-        });
-
-        Object.values(CSI_SECRETS).forEach((key) => {
+        // UI components for Longhorn storage class parameters
+        const defaultParameters = [
+          'numberOfReplicas',
+          'staleReplicaTimeout',
+          'diskSelector',
+          'nodeSelector',
+          'migratable',

```

Comment:
In the 1.3.0 codebase, `migratable` should be part of the default parameters, but it is excluded from the volumeEncryption features in `1.4.0`.
Ref: https://github.com/harvester/dashboard/blob/release-harvester-v1.3.0/pkg/harvester/edit/harvesterhci.io.storage/provisioners/driver.longhorn.io.vue#L13-L17